models/model.js

```diff
@@ -141,3 +141,13 @@ exports.addArticleComment = async (id, body) => {
     }
 }
 
+exports.fetchArticleComments = async (id) => {
+
+    await this.checkArticleExists(id)
```

Comment:
Nothin inherently wrong with this Will! but bear in mind that these models are now coupled together - any alteration one one will affect another - something to be aware of! Can always take place in the controller